src/main/java/ru/hogwarts/school/service/StudentService.java

```diff
@@ -130,7 +134,9 @@ public Collection<Student> findByAgeBetween(int max, int min) {
         return studentRepository.findByAgeBetween(max, min);
     }
 
-    private synchronized void printName(int index) {
-        System.out.println(studentRepository.findAll().get(index).getName());
+    private synchronized void printName() {
+        List<Student> students = studentRepository.findAll();
+        System.out.println(students.get(count).getName());
```

Comment:
```suggestion
        System.out.println(students.get(count % students.size()).getName());
```
потому что иначе будет выход за границы, когда будешь вызывать много раз, т.к. счётчик увеличивается

Reply:
> потому что иначе будет выход за границы, когда будешь вызывать много раз, т.к. счётчик увеличивается

Да и правда. Спасибо!